components/org.wso2.carbon.business.rules.web/pom.xml

```diff
@@ -123,6 +123,14 @@
                 </includes>
                 <targetPath>pages</targetPath>
             </resource>
+            <resource>
+                <directory>${basedir}/public/locales</directory>
+                <includes>
+                    <include>en.json</include>
+                    <include>fr.json</include>
+                </includes>
```

Comment:
Any reason to add below snippet?

 <includes>
       <include>en.json</include>
       <include>fr.json</include>
  </includes>